packages/core/src/Cardano/types/DelegationsAndRewards.ts

```diff
@@ -27,6 +28,7 @@ export interface RewardAccountInfo {
   address: RewardAccount;
   credentialStatus: StakeCredentialStatus;
   delegatee?: Delegatee;
+  dRepDelegatee?: DelegateRepresentative;
```

Comment:
This would make it easier to add some metadata in the future
```suggestion
type DRepDelegatee = { delegateRepresentative: DelegateRepresentative; }
...
    dRepDelegatee?: DRepDelegatee;
```

Reply:
done in https://github.com/input-output-hk/cardano-js-sdk/pull/1466/commits/2cafb87ae9fd6a481f0ed0b4b329e5381165375a